### PR TITLE
CVE-2017-5638

### PR DESCRIPTION
**Pull request by yerngpaii**
updated yml file answering questions.

### CODE REVIEW

File: cves/CVE-2017-5638.yml [hunk 1]
```diff
@@ -45,7 +45,12 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  Apache Struts uses the Jakarta Multiparser, and it generates an exception when parsing
```
Comment (Xetnus) [inline cves/CVE-2017-5638.yml]: What is the "Jakarta Multiparser" and what does it do? A brief explanation might help those of us not familiar with Apache Struts.

File: cves/CVE-2017-5638.yml [hunk 2]
```diff
@@ -45,7 +45,12 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  Apache Struts uses the Jakarta Multiparser, and it generates an exception when parsing 
+  the Content-Type. It raises this exception and includes the invalid data in the 
+  error message, and does this by parsing the input and treating the parsed result as an 
+  OGNL expression. This is potential for remote code execution. This affects Apache Struts 
+  2.3.5 - 2.3.31 and 2.5 - 2.5.10
```
Comment (Xetnus) [inline cves/CVE-2017-5638.yml]: I think it'd be best if you removed any references to version numbers. If we wanted to have more detail, we could just look it up later.

File: cves/CVE-2017-5638.yml [hunk 3]
```diff
@@ -119,19 +129,22 @@ discovered:
...
 
 subsystem:
   question: |
     What subsystems was the mistake in?
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
+  answer: |
+    Interceptor, input validation and error handling
```
Comment (Xetnus) [inline cves/CVE-2017-5638.yml]: This seems like a fairly long name. Could you shorten it down to just one or two words? Maybe just "interceptor" or something else more pertinent to the vulnerability?

File: cves/CVE-2017-5638.yml (hunk 2 above)
Comment (yerngpaii) [reply cves/CVE-2017-5638.yml]: noted

File: cves/CVE-2017-5638.yml (hunk 3 above)
Comment (yerngpaii) [reply cves/CVE-2017-5638.yml]: noted

File: cves/CVE-2017-5638.yml (hunk 1 above)
Comment (yerngpaii) [reply cves/CVE-2017-5638.yml]: I updated the comment to be more descriptive, Jakarta is basically a built in parser that is part of Struts and is used to parse input from HTTP POST requests.

**Pull request merged**